xml/sle_update_preparation.xml

```diff
@@ -35,9 +37,12 @@
  <sect1 xml:id="sec-update-preparation-relnotes">
   <title>Read the release notes</title>
   <para>
-   In the release notes you can find additional information on changes
-   since the previous release of &productname;. Check the release notes
-   to see whether:
+   Find the latest changes and additions in the release notes.
+  </para>
+  <para>
+   Check the release notes at <link
+    xlink:href="https://www.suse.com/releasenotes/"/> or on the installation DVD in the
+   <filename>docu</filename> directory to see whether:
```

Comment:
I think we need to rephrase this. The RN not only list the _latest_ changes but _all_ _relevant_ changes since the last release.

The latest changes you are referring to comes from the deleted para in lines 59-62 where we mention changes that did not make it into the docs. We should mention them to, but in a positive way, something along the lines of "latest additions to the documentation".

And last but not least, I think that mentioning known issues (deleted, was in line 61) are worth mentioning, too.

Reply:
rephrased this section